mrkt.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '~> 2.0'
 
-  spec.add_dependency 'faraday_middleware', '~> 0.9.1'
+  spec.add_dependency 'faraday_middleware', '~> 0.10.0'
```

Comment:
Actually we are compatible with 0.9 branch as well, se we should allow that version as well.